docs/notes.md

```diff
@@ -878,5 +881,438 @@ With these observations, we can simplify the combined \\(m\\)-party statement ab
 \end{aligned}
 \\]
 
+Arithmetic Circuit Proofs
+=========================
+
+The goal of an *arithmetic circuit proof* is for a prover to convince a verifier that a particular set of values \\(v\\) satisfy the constraints represented by the arithmetic circuit, without revealing any additional information about the values \\(v\\).
+
+The prover will use the efficient inner product protocol to do this, so we want to work towards expressing the arithmetic circuit's conditions in terms of a single inner product. The prover begins with a vector of secret values \\(v\\), and a vector of commitments to those secret values \\(V\\). The prover will send \\(V\\) to the verifier, along with the proof.
+
+Notation for arithmetic circuit proofs
+------------------------------------------
+
+In the paper, matrices are labeled as \\( \textbf{W}\_L, \textbf{W}\_R, \textbf{W}\_O, \textbf{W}\_V \\). We will keep this notation, but will note to readers not to confuse the \\(\textbf{W}\_{L,R,O,V}\\) notation for being a vector of points. 
+
+We will use the notation described in the [`notation`](index.html#notation) section of the notes. In addition, we rename one more variable from the paper, to make it clear which variables are blinding factors:
+\\[
+\begin{aligned}
+    \beta         &\xrightarrow{} \tilde{o} \\\\
+\end{aligned}
+\\]
+
+Proving statements about arithmetic circuits gates
+--------------------------------------------------
+
+One type of gate in an arithmetic circuit is a multiplication gate of fan-in 2, which takes two input values and multiplies them to get an output value. If for all of the multiplication gates in a circuit, \\(\textbf{a}\_L\\) is the vector of left inputs, \\(\textbf{a}\_R\\) is the vector of right inputs, and \\(\textbf{a}\_O\\) is the vector of outputs, then the following equation will represent the relationship between the inputs and outputs of all the multiplication gates in the circuit:
```

Comment:
So far we just have a vector of values `v`, which are the commitments to the input wires, but now there are three vectors corresponding to the left/right inputs and outputs of all of the gates.  I think it would be good to have described the connection between the circuit DAG (with input wires only at the beginning) and the R1CS setup (where the gates are all pulled apart and then re-connected by consistency equations) before defining what the left/right/output vectors are.